app/stores/MessageUnreadStore.js

```diff
@@ -120,5 +135,17 @@ let UnreadStore = assign({}, BaseStore, {
     })
 
 });
-
+function getMessagesCount(channel, oldestMessage) {
```

Comment:
Move this function to InitAction.js, we keep the specification that call ajax in "action" (maybe it is the best practice)